spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/catalog/GravitinoCatalog.java

```diff
@@ -382,6 +382,20 @@ static com.datastrato.gravitino.rel.TableChange transformTableChange(TableChange
       TableChange.RenameColumn renameColumn = (TableChange.RenameColumn) change;
       return com.datastrato.gravitino.rel.TableChange.renameColumn(
           renameColumn.fieldNames(), renameColumn.newName());
+    } else if (change instanceof TableChange.UpdateColumnPosition) {
+      TableChange.UpdateColumnPosition sparkUpdateColumnPosition =
+          (TableChange.UpdateColumnPosition) change;
+      com.datastrato.gravitino.rel.TableChange.UpdateColumnPosition gravitinoUpdateColumnPosition =
+          (com.datastrato.gravitino.rel.TableChange.UpdateColumnPosition)
+              com.datastrato.gravitino.rel.TableChange.updateColumnPosition(
+                  sparkUpdateColumnPosition.fieldNames(),
+                  transformColumnPosition(sparkUpdateColumnPosition.position()));
+      Preconditions.checkArgument(
+          !(gravitinoUpdateColumnPosition.getPosition()
+              instanceof com.datastrato.gravitino.rel.TableChange.Default),
+          "Unsupported table column position "
```

Comment:
change it to `Doesn't support alter column position without specifying position` ?